package.json

```diff
@@ -32,24 +32,26 @@
     },
     "homepage": "https://github.com/mertkahyaoglu/storybook-zeplin#readme",
     "peerDependencies": {
-        "@storybook/addons": "^6.1.10",
-        "@storybook/api": "^6.1.10",
-        "@storybook/components": "^6.1.10",
-        "@storybook/theming": "^6.1.10",
-        "react": ">=16.8.0"
+        "@storybook/addons": "^6.0.0",
```

Comment:
I believe this addon should work just fine for Storybook 6.0 versions as well. We should loose the requirements on the peer dependency if we can.